packages/terra-switch/translations/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Terra.switch.switchstatuslabel.on": "ON",
```

Comment:
Are these intended to be all capitalized? I see On/Off being used in the zeplin implementation wireframes.

Reply:
@tbiethman There is one textual reference in the API board that uses all caps when talking about states, but all visuals do not. Sorry for any confusion. They should NOT be all caps.